core/src/test/scala/unit/kafka/server/ConsumerGroupDescribeRequestsTest.scala

```diff
@@ -116,6 +115,7 @@ class ConsumerGroupDescribeRequestsTest(cluster: ClusterInstance) extends GroupC
     val timeoutMs = 5 * 60 * 1000
     val clientId = "client-id"
     val clientHost = "/127.0.0.1"
+    val authorizedOperationsInt = 328; // Integer representation of the authorized operations for this request
```

Comment:
I wonder if we should replace `328` by something like
```
      Utils.to32BitField(
        AclEntry.supportedOperations(ResourceType.CLUSTER).asScala
          .map(_.code.asInstanceOf[JByte]).asJava)
```
What do you think? We could also do it in the other place.

Reply:
done